src/frontend/AppNavigator.tsx

```diff
@@ -1,18 +1,38 @@
-import {NavigationContainer} from '@react-navigation/native';
+import {
+  NavigationContainer,
+  type NavigationContainerRef,
+} from '@react-navigation/native';
 import * as React from 'react';
 import * as SplashScreen from 'expo-splash-screen';
+import {type AppStackParamsList} from './sharedTypes/navigation';
+
 import {useSetUpInvitesListeners} from '@comapeo/core-react';
 import {RootStackNavigator} from './Navigation/Stack';
+import type Sentry from '@sentry/react-native';
 
-export const AppNavigator = ({permissionAsked}: {permissionAsked: boolean}) => {
+export const AppNavigator = ({
+  permissionAsked,
+  navigationIntegration,
+}: {
+  permissionAsked: boolean;
+  navigationIntegration:
+    | ReturnType<(typeof Sentry)['reactNavigationIntegration']>
+    | undefined;
+}) => {
+  const containerRef =
+    React.useRef<NavigationContainerRef<AppStackParamsList>>(null);
```

Comment:
should we just use the [useNavigationContainerRef](reactnavigation.org/docs/navigation-container#ref)?

Reply:
Good catch, yes. I was following some docs from somewhere but not sure where, and this function is better. I'll update.